dll/directx/wine/dsound/primary.c

```diff
@@ -459,7 +459,7 @@ LPWAVEFORMATEX DSOUND_CopyFormat(LPCWAVEFORMATEX wfex)
 
 HRESULT primarybuffer_SetFormat(DirectSoundDevice *device, LPCWAVEFORMATEX wfex)
 {
-	HRESULT err = DSERR_BUFFERLOST;
+	HRESULT err = S_OK;
```

Comment:
There's `DS_OK` which is the same as S_OK but strictly exclusive to DirectSound code. See https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=%23define+DS_OK

Reply:
Indeed, it exists. And seems to be more logically suitable here. But on my opinion, I'd keep the current one, at least to have less diffs with Wine, moreover if these values have the same effect. I think even if we forked dsound from Wine already, it still makes no sense to add additional unneeded diffs, when we can avoid using them.